src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -325,6 +340,26 @@ private void decodeOtherParameter(Position position, int id, ByteBuf buf, int le
             case 241:
                 position.set(Position.KEY_OPERATOR, readValue(buf, length, false));
                 break;
+            case 246:
+                if (readValue(buf, length, false) == 1) {
+                    position.set(Position.KEY_ALARM, Position.ALARM_TOW);
+                }
+                break;
+            case 247:
+                if (readValue(buf, length, false) >= 1) {
```

Comment:
Why is this one different from other (`>=` instead of `==`).

Reply:
according to docs

    1 – Crash Detected
    2 – Crash Trace Record, (begins 5 sec before crash, and ends 5
      sec after crash